.github/workflows/deploy/chocolatey/asyncapi-modelina.nuspec

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>asyncapi</id>
```

Comment:
You will have to change the id to something else as this won't work. Maybe modelina can be the id.

Reply:
Changing 👍